### PR TITLE
RSWEB-9084: update icon fonts

### PR DESCRIPTION
**Pull request by brantdarilek**

### CODE REVIEW

File: fonts/css/rswebfonts-ie7-codes.css [hunk 1]
```diff
@@ -222,7 +226,7 @@
 .icon-srv-content-management { *zoom: expression( this.runtimeStyle['zoom'] = '1', this.innerHTML = '&#xe9ab;&nbsp;'); }
 .icon-srv-data-storage { *zoom: expression( this.runtimeStyle['zoom'] = '1', this.innerHTML = '&#xe9ad;&nbsp;'); }
 .icon-srv-database { *zoom: expression( this.runtimeStyle['zoom'] = '1', this.innerHTML = '&#xe9ae;&nbsp;'); }
-.icon-srv-design-build { *zoom: expression( this.runtimeStyle['zoom'] = '1', this.innerHTML = '&#xe9af;&nbsp;'); }
+.icon-srv-design-build-2 { *zoom: expression( this.runtimeStyle['zoom'] = '1', this.innerHTML = '&#xe9af;&nbsp;'); }
```
Comment (amberRrucker) [inline fonts/css/rswebfonts-ie7-codes.css]: @brantdarilek We have to have both icon-srv-design-build and icon-srv-design-build-2 here to support some legacy icons.

File: fonts/demo.html [hunk 2]
```diff
@@ -634,7 +640,7 @@
         <div title="Code: 0xe9ab" class="the-icons span3"><i class="demo-icon icon-srv-content-management">&#xe9ab;</i> <span class="i-name">icon-srv-content-management</span><span class="i-code">0xe9ab</span></div>
         <div title="Code: 0xe9ad" class="the-icons span3"><i class="demo-icon icon-srv-data-storage">&#xe9ad;</i> <span class="i-name">icon-srv-data-storage</span><span class="i-code">0xe9ad</span></div>
         <div title="Code: 0xe9ae" class="the-icons span3"><i class="demo-icon icon-srv-database">&#xe9ae;</i> <span class="i-name">icon-srv-database</span><span class="i-code">0xe9ae</span></div>
-        <div title="Code: 0xe9af" class="the-icons span3"><i class="demo-icon icon-srv-design-build">&#xe9af;</i> <span class="i-name">icon-srv-design-build</span><span class="i-code">0xe9af</span></div>
+        <div title="Code: 0xe9af" class="the-icons span3"><i class="demo-icon icon-srv-design-build-2">&#xe9af;</i> <span class="i-name">icon-srv-design-build-2</span><span class="i-code">0xe9af</span></div>
```
Comment (amberRrucker) [inline fonts/demo.html]: same here

File: fonts/css/rswebfonts-ie7-codes.css (hunk 1 above)
Comment (brantdarilek) [inline fonts/css/rswebfonts-ie7-codes.css]: @amberRrucker 
This shouldn't matter as its only part of the css generated by fontello and never gets consumed by zoolander or www. The important part is here https://github.com/brantdarilek/zoolander/blob/rsweb-9084-icon-font/styleguide/_themes/global/scss/rsweb-font-style.scss#L359 which is were we specify the icon in our styles which actually do get consumed.

File: fonts/demo.html (hunk 2 above)
Comment (brantdarilek) [inline fonts/demo.html]: See above note.

**Pull request merged**